tools/speech_data_explorer/data_explorer.py

```diff
@@ -1,846 +1,1044 @@
-# Copyright (c) 2020, NVIDIA CORPORATION.  All rights reserved.
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#     http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-
-import argparse
-import base64
-import csv
-import datetime
-import difflib
-import io
-import json
-import math
-import operator
-import os
-import pickle
-from collections import defaultdict
-from os.path import expanduser
```

Comment:
Please integrate Ante's updates back (to support audio_base_path)

Reply:
done